src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -221,18 +245,20 @@ export const CustomHeadings: Story = {
         view="form"
         onChange={() => {}}
         onSubmit={() => {}}
-        title="Custom string heading"
-        confirmationHeading={defaultConfirmationHeading}
-        confirmationText={defaultConfirmationText}
+        title="String heading"
```

Comment:
Related to the comment above. The next example has the h4 element for `title` and this one has  a string 👍